backend/core/migrations/0100_auto_20220919_0325.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 3.2.9 on 2022-09-19 03:25
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("core", "0099_auto_20220830_0405"),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name="ingredient",
+            name="position",
```

Comment:
is this backwards compatible?

Like we switch to lexicographical sorting the existing position floats won't work right?

Reply:
floats are lexicographically sortable when converting to strings